project-base/storefront/components/Pages/App/Loaders.tsx

```diff
@@ -10,6 +12,8 @@ export const Loaders = () => {
     useAuthLoader();
     usePageLoader();
     useReloadCart();
+    useRefetchComparedProducts();
+    useRefetchWishedProducts();
```

Comment:
I see these two are quite similar, they both listen on the same broadcast event `refetchProductList`. Couldn't we have just one `useRefetchProductList`? Right now when you add/remove wished product for example, doesn't it trigger this event handler twice?

If these cannot be merged into single handler, I would maybe consider splitting the event into two separate ones (eg. `refetchWishedProducts` and `refetchComparedProducts`).

Reply:
fixed :)